action.yml

```diff
@@ -13,8 +13,8 @@ inputs:
     default: ${{ github.token }}
     required: false
   debug:
-    description: Whether to tell the GitHub client to log details of its requests
-    default: false
+    description: Whether to tell the GitHub client to log details of its requests. true or false. Default is to run in debug mode when the GitHub Actions step debug logging is turned on.
+    default: ${{ (runner.debug == '1' && 'true') || 'false' }}
```

Comment:
Why does `runner.debug == '1'` not work?

Reply:
It does, but I just didn't think of it 🤦.

Reply:
When `runner.debug` is not set, will `runner.debug == '1'` evaluate to `''` or `false`?

Reply:
ok, it is false https://github.com/TWiStErRob/github-actions-test/actions/runs/4620614546/jobs/8170965701#step:4:20